src/main/kotlin/frc/robot/robotstate/RobotCommands.kt

```diff
@@ -37,14 +40,17 @@ val turretAngleToHub: Angle
 val swerveCompensationAngle
     get() = drive.rotation + Rotation2d(angleFromRobotHub - turretAngleToHub)
 
+val BallGlobalPoses
+    get() = Array<Pose2d>(ballPoses.size) { i -> drive.pose + ballPoses[i].toPose2d().toTransform() }
```

Comment:
```suggestion
    get() = ballPoses.map { drive.pose + it.toPose2d().toTransform() }
```
Also: `ballPoses` should be called `robotRelativeBallPoses`, and this one should be `globalBallPoses`.

Reply:
I think it's the wrong commit number

Reply:
ok but also added some changes to make it better 4fa15d2e8e20a888186c7e2526ba93d40b513290